### DISCUSSION TITLE
How command delete line?

### DISCUSSION THREAD

**Discussion by igson** [LunarVim/LunarVim#3906] [General]
Is there a command to delete the line? If not, can you create your own command?

**Comment by brneor** [comment]
Vim's native command `dd`.

**Reply by igson** [reply]
> Vim's native command `dd`.

I know. But there is a way to map to ctrl+d very similar to what happens in eclipse?

**Reply by brneor** [reply]
`lvim.keys.normal_mode["<C-d>"] = "dd"` or `vim.keymap.set("n", "<C-d>", "dd" )`

**Reply by igson** [reply]
Thanks a lot
